recipes/quatradis/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -x -e
+
+pip install -r requirements.txt
```

Comment:
You should not install from `requirements.txt`. You need to instead list all dependencies in the `run:` section of the recipe.